client/src/pages/AddReview/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+// https://www.npmjs.com/package/react-typeform-embed
+import { ReactTypeformEmbed } from 'react-typeform-embed';
+
+import { Button } from '../../components';
+import urls from './urls';
+
+import './style.css';
+
+let typeformEmbed = null;
```

Comment:
watch out here, having this variable on a module level means it will be shared by all instances of the `AddReview` component. So if your application creates 2 `AddReview`s, the second one will overwrite the first one, which could lead to nasty bugs (in this case, if you clicked on the first component's button it would open the form of the second component). To prevent this, the variable needs to be declared within the scope of the component, so that each component has its own instance:
```js
const AddReview = ({ type, sitspotId }) => {
    let typeformEmbed = null;
    return (
    ...
    );
}
```

Reply:
Yes thanks noted, ill fix it now

Reply:
Fixed